package.json

```diff
@@ -54,11 +54,11 @@
     "@langchain/mistralai": "^0.1.1",
     "@types/graphql-upload": "^8.0.11",
     "amqplib": "^0.10.4",
-    "chromadb": "^1.10.1",
+    "chromadb": "^3.0.17",
```

Comment:
check if this version is compatible with the cloud db, and if we need to migrate it to a newer version and how.